### PR TITLE
docs: about create serial and parallel node on dashboard

### PR DESCRIPTION
**Pull request by STRRL**
Signed-off-by: STRRL <str_ruiling@outlook.com>

### CODE REVIEW

**Review by g1eny0ung — APPROVED** [review]
LGTM. And some conflicts need to be resolved

File: i18n/zh/docusaurus-plugin-content-docs/current/run-chaos-experiments-in-serial-or-parallel.md [hunk 1]
```diff
@@ -122,3 +122,23 @@ spec:
...
+
+![Create Serial Node On Dashboard](./img/create-serial-node-on-dashboard.png)
+
+### 并行节点
+
+你可以创建一个 `Parallel` 节点，然后在其下创建子节点。
+
+![Create Parallel Node on Dashboard](./img/create-parallel-node-on-dashboard.png))
+
+### 嵌套串行与并行节点
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/run-chaos-experiments-in-serial-or-parallel.md:140]:
```suggestion
### 创建串行节点

Chaos Dashboard 会创建一个预定义的串行节点，名为 `entry`。因此，当在 Chaos Dashboard 中创建串行节点的 Workflow 时，默认会在 `entry` 下创建流程。

![Create Serial Node On Dashboard](./img/create-serial-node-on-dashboard.png)

### 创建并行节点

你可以在 Chaos Dashboard 中创建一个并行节点 `Parallel`，然后在该节点下创建子节点。

![Create Parallel Node on Dashboard](./img/create-parallel-node-on-dashboard.png)

### 嵌套串行节点与并行节点
```

File: i18n/zh/docusaurus-plugin-content-docs/version-2.1.1/run-chaos-experiments-in-serial-or-parallel.md [hunk 2]
```diff
@@ -122,3 +122,23 @@ spec:
...
+
+![Create Serial Node On Dashboard](./img/create-serial-node-on-dashboard.png)
+
+### 并行节点
+
+你可以创建一个 `Parallel` 节点，然后在其下创建子节点。
+
+![Create Parallel Node on Dashboard](./img/create-parallel-node-on-dashboard.png))
+
+### 嵌套串行与并行节点
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.1.1/run-chaos-experiments-in-serial-or-parallel.md:140]:
```suggestion
### 创建串行节点

Chaos Dashboard 会创建一个预定义的串行节点，名为 `entry`。因此，当在 Chaos Dashboard 中创建串行节点的 Workflow 时，默认会在 `entry` 下创建流程。

![Create Serial Node On Dashboard](./img/create-serial-node-on-dashboard.png)

### 创建并行节点

你可以在 Chaos Dashboard 中创建一个并行节点 `Parallel`，然后在该节点下创建子节点。

![Create Parallel Node on Dashboard](./img/create-parallel-node-on-dashboard.png)

### 嵌套串行节点与并行节点
```

File: docs/run-chaos-experiments-in-serial-or-parallel.md [hunk 3]
```diff
@@ -122,3 +122,23 @@ The above commands claimed a parallel node named `parallel-of-2-chaos`. This mea
...
+### Parallel Node
+
+You could create an `Parallel` node, then create sub-nodes under it.
+
+![Create Parallel Node on Dashboard](./img/create-parallel-node-on-dashboard.png))
+
+### Nested Serial and Parallel Nodes
+
+You could also combine any serial and parallel as you want.
+
```
Comment (en-jin19) [inline docs/run-chaos-experiments-in-serial-or-parallel.md:143]:
```suggestion
### 创建并行节点

你可以在 Chaos Dashboard 中创建一个并行节点 `Parallel`，然后在该节点下创建子节点。

![Create Parallel Node on Dashboard](./img/create-parallel-node-on-dashboard.png)

### 嵌套串行节点与并行节点

### Create serial nodes

Chaos Dashboard creates a predefined serial node called `entry`. Therefore, when creating Workflow with serial nodes using Chaos Dashboard, the Workflow is created under `entry` by default.

![Create Serial Node On Dashboard](./img/create-serial-node-on-dashboard.png)

### Create parallel nodes

You can create a parallel node `Parallel` and create sub-nodes under `Parallel`.

![Create Parallel Node on Dashboard](./img/create-parallel-node-on-dashboard.png)

### Nest serial and parallel nodes

You can create more complex processes by nesting serial and parallel nodes together.

```

File: versioned_docs/version-2.1.1/run-chaos-experiments-in-serial-or-parallel.md [hunk 4]
```diff
@@ -122,3 +122,23 @@ The above commands claimed a parallel node named `parallel-of-2-chaos`. This mea
...
+
+### Parallel Node
+
+You could create an `Parallel` node, then create sub-nodes under it.
+
+![Create Parallel Node on Dashboard](./img/create-parallel-node-on-dashboard.png))
+
+### Nested Serial and Parallel Nodes
+
+You could also combine any serial and parallel as you want.
```
Comment (en-jin19) [inline versioned_docs/version-2.1.1/run-chaos-experiments-in-serial-or-parallel.md:142]:
```suggestion
## Create Workflow with Serial or Parallel nodes using Chaos Dashboard

### Create serial nodes

Chaos Dashboard creates a predefined serial node called `entry`. Therefore, when creating Workflow with serial nodes using Chaos Dashboard, the Workflow is created under `entry` by default.

![Create Serial Node On Dashboard](./img/create-serial-node-on-dashboard.png)

### Create parallel nodes

You can create a parallel node `Parallel` and create sub-nodes under `Parallel`.

![Create Parallel Node on Dashboard](./img/create-parallel-node-on-dashboard.png)

### Nest serial and parallel nodes

You can create more complex processes by nesting serial and parallel nodes together.
```

**Review by en-jin19 — CHANGES_REQUESTED** [review]
I have language reviewed the docs, please take a look, thanks!

File: i18n/zh/docusaurus-plugin-content-docs/current/run-chaos-experiments-in-serial-or-parallel.md [hunk 5]
```diff
@@ -122,3 +122,23 @@ spec:
 并行节点执行时，会同时执行 `children` 中所有声明的任务。
 
 并行节点同样存在 `deadline` 可选字段，类似于串行节点，目的是限制整个并行流程的最长执行时间。若达到了这个时间，其下属的子节点将会被停止。若所有子节点先于 `deadline` 完成了行为，并行节点会立刻被标记为完成，`deadline` 没有任何影响。
+
+## 在 Chaos Dashboard 中创建串行或并行节点的 Workflow
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/run-chaos-experiments-in-serial-or-parallel.md:126]:
```suggestion
## 通过 Chaos Dashboard 创建串行或并行节点的 Workflow
```

File: i18n/zh/docusaurus-plugin-content-docs/version-2.1.1/run-chaos-experiments-in-serial-or-parallel.md [hunk 6]
```diff
@@ -122,3 +122,23 @@ spec:
 并行节点执行时，会同时执行 `children` 中所有声明的任务。
 
 并行节点同样存在 `deadline` 可选字段，类似于串行节点，目的是限制整个并行流程的最长执行时间。若达到了这个时间，其下属的子节点将会被停止。若所有子节点先于 `deadline` 完成了行为，并行节点会立刻被标记为完成，`deadline` 没有任何影响。
+
+## 在 Chaos Dashboard 中创建串行或并行节点的 Workflow
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.1.1/run-chaos-experiments-in-serial-or-parallel.md:126]:
```suggestion
## 通过 Chaos Dashboard 创建串行或并行节点的 Workflow
```

File: docs/run-chaos-experiments-in-serial-or-parallel.md [hunk 7]
```diff
@@ -122,3 +122,23 @@ The above commands claimed a parallel node named `parallel-of-2-chaos`. This mea
 When Chaos Mesh executes parallel nodes, all tasks claimed in `children` are executed simultaneously.
 
 Similar to serial nodes, the optional field `deadline` is also available in parallel nodes to limit the maximum execution time of the entire parallel process. If this time is reached, the sub-nodes are stopped. If all sub-nodes finish their work before `deadline` time, parallel nodes are immediately marked as completed and `deadline` is not affected.
+
+## Create Workflow with Serial or Parallel nodes using Chaos Dashboard
```
Comment (en-jin19) [inline docs/run-chaos-experiments-in-serial-or-parallel.md:126]:
```suggestion
## Create a workflow with serial or parallel nodes using Chaos Dashboard
```

**Comment by STRRL** [comment]
Updated! PTAL

@en-jin19

**Review by en-jin19 — APPROVED** [review]
LGTM

**Pull request merged**